servers/rendering/rendering_device_commons.h

```diff
@@ -897,6 +897,7 @@ class RenderingDeviceCommons : public Object {
 		// If not supported, a fragment shader with only side effects (i.e., writes  to buffers, but doesn't output to attachments), may be optimized down to no-op by the GPU driver.
 		SUPPORTS_FRAGMENT_SHADER_WITH_ONLY_SIDE_EFFECTS,
 		SUPPORTS_BUFFER_DEVICE_ADDRESS,
+		SUPPORTS_WIDE_LINES
```

Comment:
Add trailing comma (for cleaner diffs in the future):

```suggestion
		SUPPORTS_WIDE_LINES,
```